main.go

```diff
@@ -140,10 +140,12 @@ func getWatchNamespace() string {
 	// An empty value means the operator is running with cluster scope.
 	var watchNamespaceEnvVar = "WATCH_NAMESPACE"
 
-	ns, found := os.LookupEnv(watchNamespaceEnvVar)
-	if !found {
-		setupLog.Info("unable to get WatchNamespace, "+
-			"the manager will watch and manage resources in all namespaces",
+	ns, _ := os.LookupEnv(watchNamespaceEnvVar)
+
+	// Check if operator is running as cluster scoped
+	if ns == "" {
```

Comment:
I think that

```
len(ns) > 0
```

is better to verify empty vars.

Reply:
Changed